ports/stm/peripherals/stm32f4/stm32f405xx/periph.c

```diff
@@ -29,55 +29,57 @@
 #include "peripherals/pins.h"
 #include "peripherals/periph.h"
 
-// I2C
+I2C_TypeDef * mcu_i2c_banks[I2C_BANK_ARRAY_LEN] = {I2C1, I2C2, I2C3};
 
-I2C_TypeDef * mcu_i2c_banks[3] = {I2C1, I2C2, I2C3};
-
-const mcu_periph_obj_t mcu_i2c_sda_list[4] = {
+const mcu_periph_obj_t mcu_i2c_sda_list[I2C_SDA_ARRAY_LEN] = {
     PERIPH(1, 4, &pin_PB07),
     PERIPH(1, 4, &pin_PB09),
     PERIPH(2, 4, &pin_PB11),
     PERIPH(3, 4, &pin_PC09),
+    PERIPH(2, 4, &pin_PF00),
+    PERIPH(2, 4, &pin_PH05),
+    PERIPH(3, 4, &pin_PH08),
 };
```

Comment:
I'm surprised to see new options for the SDA pin -- expected?  (I did not do any meaningful review on these "periph.[ch]" files)

Reply:
I added the original file for the F405 very early in the port process - back then, we had less clear distinction between the STM32 line and the package. So when I originally put these and the pin files together, I didn't fill out the later GPIO banks because we didn't support the packages they appeared in. We now support new packages with minimal effort, so these represent additional timer and peripheral opportunities that should be included.